util/create_dashboard_html.py

```diff
@@ -101,6 +102,27 @@ def get_ontology_order(data):
     return order
 
 
+def reorder_status(data):
+    """
+    """
+    s_pass = []
+    s_info = []
+    s_warn = []
+    s_error = []
+    for ont in data:
+        if ont.get('summary'):
+            if ont['summary']['status'] == 'ERROR':
+                s_error.append(ont)
+            elif ont['summary']['status'] == 'WARN':
```

Comment:
I know this change makes some sense, but it was not agreed upon by OBO Ops. Its always a bit political with these kinds of decisions. For now just separate between s_pass and s_error, and whenever something has an error, put it in error, else put it in pass. If you feel this more fine grained solution is better, you will have to make an issue and present this at the next call!

Reply:
Ok, I'll split only between error and pass.

Reply:
```python
ORDERING = {"ERROR": 0, "WARN": 1, "INFO": 2, "PASS": 3}
data = sort(data, key=lambda ont: ORDERING[ont['summary']['status']]
```

Reply:
Thank you, Charlie! Will it handle when there isn't a summary key?

Reply:
adding an `if` would do the trick

```python
ORDERING = {"ERROR": 0, "WARN": 1, "INFO": 2, "PASS": 3}
return sorted(
    data, 
    key=lambda ont: ORDERING[ont['summary']['status']] if 'summary' in ont else 5
)

Reply:
Thanks, Charlie!